src/main/java/edu/umich/tl/Course.java

```diff
@@ -0,0 +1,34 @@
+package edu.umich.tl;
+
+public class Course {
+	private int id;
+	private String courseName;
+	private String startDate;
```

Comment:
All i need to check whether the start_date and end_date fields is not null.So i just left it as string than date format.